lib/drivers/dexcom/dexcomDriver.js

```diff
@@ -1272,33 +1384,33 @@ module.exports = function (config) {
         sessionInfo,
         progress,
         cfg.groupId,
-        function (err, result) {
-        if (err) {
-          debug(err);
-          progress(100);
-          return cb(err, data);
-        } else {
+        (err) => {
+          if (err) {
+            debug(err);
+            progress(100);
+            return cb(err, data);
+          }
           progress(100);
           return cb(null, data);
-        }
-      });
+        }, 'dataservices',
+      ); // TODO: REMOVE THIS! (switching to platform now would cause dupes)
```

Comment:
Should this TODO be addressed?

Reply:
Oh yes, good catch! After changing this back, I just discovered that Jellyfish has a validation check that leads to the following error:

`food.carbs should be a number, got[undefined]`

This should be changed, as the updated `food` type on platform uses a different data structure. We never used the old `food` structure that Jellyfish is trying to validate. I just created https://tidepool.atlassian.net/browse/BACK-1692 to track this.

Related question: If the Dexcom API uses platform instead of Jellyfish to upload `food` data, what happens with deduplication when someone uploads their receiver through Jellyfish?

Reply:
It won't be deduplicated. Same issue we have with multiple sources anywhere in Tidepool.